app/src/main/java/org/fossasia/phimpme/gallery/activities/SingleMediaActivity.java

```diff
@@ -1153,8 +1154,13 @@ public void onClick(DialogInterface dialog, int which) {
                     slideshow=true;
                     int intValue = Integer.parseInt(value);
                     SLIDE_SHOW_INTERVAL = intValue * 1000;
-                    hideSystemUI();
-                    handler.postDelayed(slideShowRunnable, SLIDE_SHOW_INTERVAL);
+                    if(SLIDE_SHOW_INTERVAL > 1000) {
```

Comment:
shouldn't this be > 2000 if you are saying 2 sec is the minimum duration?

Reply:
@mayank8318 as the user is allowed only to enter integer values hence by going with 2 sec minimum then it should be >=2000, not >2000 as it will not work for 2 sec also.

Reply:
Hence I have added for the greater than 1 sec :+1:

Reply:
oh yes didn't see that it didn't have '=' there.